src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentStatusService.scala

```diff
@@ -91,7 +105,7 @@ class ConsignmentStatusService(
 }
 
 object ConsignmentStatusService {
-  val validConsignmentTypes: List[String] = List("Series", "TransferAgreement", "Upload", "ClientChecks", "ConfirmTransfer", "Export")
+  val validConsignmentTypes: List[String] = List("Series", "TransferAgreement", "Upload", "ClientChecks", "ClosureMetadata", "DescriptiveMetadata", "ConfirmTransfer", "Export")
```

Comment:
I'm going to raise a new JIRA ticket about consolidating the consignment and file status types /values in to a set of case objects, instead of hard coding these strings in multiple places in the code.